### PR TITLE
[Account] `az account alias`: GA release

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
---
GA of "az account alias" https://github.com/Azure/azure-cli/issues/21937
close https://github.com/Azure/azure-cli/issues/21937

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Account

File: src/account/azext_account/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {
-    "azext.isExperimental": true,
```
Comment (zhoxing-ms) [inline src/account/azext_account/azext_metadata.json:2]: We need to confirm whether we only need GA a command group `az account alias` or GA the whole extension

Comment (joergjo) [reply src/account/azext_account/azext_metadata.json:2]: For the [scenario I have in mind](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription-enterprise-agreement?tabs=azure-cli), az account alias is sufficient.

Comment (zhoxing-ms) [reply src/account/azext_account/azext_metadata.json:2]: OK, thanks~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**